region/brinstar/green/Early Supers Room.json

```diff
@@ -457,7 +501,10 @@
         "leaveShinecharged": {
           "framesRemaining": 40
         }
-      }
+      },
+      "unlocksDoors": [{"types": ["ammo"], "requires": []}],
+      "note": "Use the runway on the screen above to gain a shinecharge near the edge of the runway.",
+      "devNote": "This starts at node 2 to be able to unlock the door if needed."
```

Comment:
This, and the one below, needs to be moved to start at 2 and have reqs added to get up to 4.

Reply:
Done. Yeah, I had it like that before and then was thinking it would be ok, since if you could get to 4 in the first place then you could get up there again. But there could be scenarios where that might not be true (e.g. if 4 were your starting location) so it's safer to do it this way like you suggest.